levels/bbh/header.h

```diff
@@ -96,7 +96,7 @@ extern const Gfx bbh_seg7_dl_0701FFE8[];
 extern const Gfx bbh_seg7_dl_070202F0[];
 extern const Gfx bbh_seg7_dl_070206F0[];
 extern const Collision bbh_seg7_collision_level[];
-extern const u8 bbh_seg7_rooms[];
+extern const RoomData bbh_seg7_rooms[];
```

Comment:
vanilla levels are changed when modding, so changing types like this is a little risky. in general keep vanilla levels unchanged so that people dont get level conflicts